### PR TITLE
build: use c8 for coverage

### PR DESCRIPTION
**Pull request by JustinBeckwith**

### CODE REVIEW

**Comment by bcoe** [comment]
@nolanmar511 is this safe to land with the circleci issues?

File: package.json [hunk 1]
```diff
@@ -52,6 +51,7 @@
     "@types/pretty-ms": "^4.0.0",
     "@types/sinon": "^7.0.12",
     "@types/tmp": "0.1.0",
+    "c8": "^7.0.0",
     "codecov": "^3.0.0",
```
Comment (nolanmar511) [inline package.json]: I assume codecov is no longer needed. Could this and codecov.yaml be removed?

Comment (bcoe) [reply package.json]: @nolanmar511 codecov is the bin used to upload coverage results to the upstream service we use, so this is still needed (nyc and c8 are what collect the coverage locally).

Comment (nolanmar511) [reply package.json]: Okay! Thanks for the clarification!

**Comment by nolanmar511** [comment]
I removed the CircleCI tests since they've been redundant since the Kokoro tests were added. 

All checks should pass now, and this should be safe to merge.

**Comment by bcoe** [comment]
@nolanmar511 thank you :clap:

**Pull request merged**